packages/commons-ui-core/src/RichTypography/index.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import RichTypography from ".";
+
+export default {
+  title: "Component/RichTypography",
+  argTypes: {},
+};
+
+function Template() {
+  return <RichTypography> This is an example</RichTypography>;
+}
+
+export const Default = Template.bind({});
+
+Default.args = {};
```

Comment:
Make the test editable so that a user can try different content and see how it’s rendered.